Modern/utilities/src/test/java/org/bonej/utilities/SharedTableTest.java

```diff
@@ -144,13 +144,13 @@ public void testRepeatingHeaderAndLabelAddsARow() {
 		SharedTable.add("Image", "Value", 1.0);
 		SharedTable.add("Image", "Run", 2);
 
-		final Table<DefaultColumn<String>, String> table = SharedTable.getTable();
+		final Table<DefaultColumn<Double>, Double> table = SharedTable.getTable();
 		assertEquals(
 			"Adding data to the same column, to the row with the same label, should create a new row",
 			2, table.getRowCount());
-		assertEquals("Values in wrong order, older should be first", "1", table.get(
+		assertEquals("Values in wrong order, older should be first", new Double(1), table.get(
 			"Run").get(0));
-		assertEquals("Values in wrong order, older should be first", "2", table.get(
+		assertEquals("Values in wrong order, older should be first", new Double(2), table.get(
 			"Run").get(1));
```

Comment:
Could use `Double.getDoubleValue()` instead of boxing the literal

Reply:
Good idea and I prefer not to instantiate an object if I can avoid it, but the polymorphism  assertEquals(String, double, double) is deprecated.

Reply:
It's the version without the final error parameter that's deprecated. (String, double, double, doulbe) works.

Reply:
OK, sorted.